app/models/user.rb

```diff
@@ -0,0 +1,14 @@
+class User < ActiveRecord::Base
+  before_create :set_auth_token
```

Comment:
@anymoto I noticed you already set a unique index on this column (:+1:). Should we add a validation for the edge case where a token already exists for a different user?